### PR TITLE
Add missing localisation for settings enums and mods tooltips

### PR DESCRIPTION
**Pull request by its5Q**
I've added localisation support for some settings labels and enums, so the settings are 99% localised.

### CODE REVIEW

File: osu.Game/Localisation/CommonStrings.cs [hunk 1]
```diff
@@ -99,6 +99,31 @@ public static class CommonStrings
...
+
+        /// <summary>
+        /// "Full"
+        /// </summary>
+        public static LocalisableString Full => new TranslatableString(getKey(@"full"), @"Full");
+
+        /// <summary>
+        /// "Off"
+        /// </summary>
+        public static LocalisableString Off => new TranslatableString(getKey(@"off"), @"Off");
```
Comment (bdach) [inline osu.Game/Localisation/CommonStrings.cs:125]: Unconvinced that strings like this are going to commonise very well. Some languages have a notion of gendered nouns, which also sometimes causes adjectives to have gendered forms, too, and so "off" becomes context-sensitive. French and Polish do this.

File: osu.Game/Overlays/Music/MusicKeyBindingHandler.cs [hunk 2]
```diff
@@ -74,7 +74,7 @@ public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
                                 break;
 
                             case PreviousTrackResult.Previous:
-                                onScreenDisplay?.Display(new MusicActionToast(GlobalActionKeyBindingStrings.MusicPrev, e.Action));
+                                onScreenDisplay?.Display(new MusicActionToast("Test", e.Action));
```
Comment (bdach) [inline osu.Game/Overlays/Music/MusicKeyBindingHandler.cs:77]: Surely accidental / unintended change?

Comment (its5Q) [reply osu.Game/Overlays/Music/MusicKeyBindingHandler.cs:77]: Yeah, sorry, was trying to figure out why some strings can't be localized accidentally, will fix

File: osu.Game/Localisation/CommonStrings.cs (hunk 1 above)
Comment (its5Q) [reply osu.Game/Localisation/CommonStrings.cs:125]: I see, yeah, you are right, I will move it to separate localisation classes.

**Comment by its5Q** [comment]
Moved "Off" and "Full" to separate classes and reverted unintentional change.

File: osu.Game/Localisation/DiscordRichPresenceModeStrings.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+        /// <summary>
+        /// "Hide identifiable information"
+        /// </summary>
+        public static LocalisableString HideIdentifiableInformation => new TranslatableString(getKey(@"hide_identifiable_information"), @"Hide identifiable information");
+
+        /// <summary>
+        /// "Full"
+        /// </summary>
+        public static LocalisableString Full => new TranslatableString(getKey(@"rich_presence.full"), @"Full");
```
Comment (peppy) [inline osu.Game/Localisation/DiscordRichPresenceModeStrings.cs:20]: What are the prefixes inside of `getKey` here?

File: osu.Game/Localisation/ManiaSettingsSubsectionStrings.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+namespace osu.Game.Localisation
+{
+    public static class ManiaSettingsSubsectionStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.ManiaSettingsSubsection";
+
+        /// <summary>
+        /// "Scrolling direction"
+        /// </summary>
+        public static LocalisableString ScrollingDirection => new TranslatableString(getKey(@"scrolling_direction"), @"Scrolling direction");
```
Comment (peppy) [inline osu.Game/Localisation/ManiaSettingsSubsectionStrings.cs:15]: Reword to "Scroll direction" to match "Scroll speed".

File: osu.Game/Localisation/OsuConfineMouseModeStrings.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class OsuConfineMouseModeStrings
```
Comment (peppy) [inline osu.Game/Localisation/OsuConfineMouseModeStrings.cs:8]: This should not be its own file. See other settings localisations.

File: osu.Game/Localisation/ManiaSettingsSubsectionStrings.cs [hunk 6]
```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class ManiaSettingsSubsectionStrings
```
Comment (peppy) [inline osu.Game/Localisation/ManiaSettingsSubsectionStrings.cs:8]: This should not be in the `osu.Game` project.

File: osu.Game/Localisation/ModSelectHotkeyStyleStrings.cs [hunk 7]
```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class ModSelectHotkeyStyleStrings
```
Comment (peppy) [inline osu.Game/Localisation/ModSelectHotkeyStyleStrings.cs:8]: This should not be its own file. See other settings localisations.

File: osu.Game/Localisation/PlayfieldBorderStyleStrings.cs [hunk 8]
```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class PlayfieldBorderStyleStrings
```
Comment (peppy) [inline osu.Game/Localisation/PlayfieldBorderStyleStrings.cs:8]: This should not be its own file. See other settings localisations.

etc.

File: osu.Game/Localisation/CommonStrings.cs (hunk 1 above)
Comment (peppy) [inline osu.Game/Localisation/CommonStrings.cs:125]: I don't think this is going to work. These words are contextual and hard to translate in such a generic way in many languages (japanese as one example).

**Review by peppy — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
Please split this PR out into one per ruleset / area. Don't mix common strings additions with other areas if possible, as they require special attention.

File: osu.Game/Localisation/DiscordRichPresenceModeStrings.cs (hunk 3 above)
Comment (its5Q) [reply osu.Game/Localisation/DiscordRichPresenceModeStrings.cs:20]: I could've seen something like that in other string classes, or just misread something, either way, there should not be any prefixes, as getKey already adds one.

**Comment by its5Q** [comment]
Thanks for the feedback, I will close this PR for now, and open a separate PR for just settings localisation, addressing all the issues above.

**Pull request closed without merging**